### DISCUSSION TITLE
My students pack got approved but benefits tabs are not showing, its showing will take time, but its already 3 days now

### DISCUSSION THREAD

**Discussion by karthik558** [community/community#138995] [GitHub Education] [answered]
### Hi everyone,

Dear sir,

My students pack got approved but benefits tabs are not showing, its showing will take time, but its already 3 days now.

Kindly check the issue and grant me access to the students benefits asap.

Regards,
KARTHIK LAL

**Comment by davevad93** [comment, marked as answer]
Hi @karthik558 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @karthik558  was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by karthik558** [reply]
Ok sir, thanks for the info

**Reply by davevad93** [reply]
You're welcome @karthik558 👍
